README.md

```diff
@@ -145,3 +145,28 @@ Pretty-printed JSON metadata listing
 Query offset(s) by timestamp(s)
 
     $ kafkacat -b mybroker -Q -t mytopic:3:2389238523  mytopic2:0:18921841
+
+# Running in docker
+
+There is no official image build yet but you may run `docker build -t kafkacat .` or use for example `solsson/kafkacat@sha256:3075a6ca2d8431910cf01af13acd2ef9542c8ed48547733eb5c4321364ef0b66`.
+
+Example:
```

Comment:
Personally I'd find the example easier to follow if it avoided detaching kafkacat instances. It's closer to how you 'd run kafkacat if it was locally installed, no need to call `docker logs`.